metalinter.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Linters": {
+	      "errcheck": "errcheck -abspath -ignore 'Close|Write|Serve' {path}:PATH:LINE:COL:MESSAGE",
```

Comment:
`gometalinter` splits this string on the first ":". Is there a way we can use a different string for that? 

Also, I've currently turned off "errcheck" because it's pretty inefficient when called from metalinter.  I've kept this line to preserve our checks for when metalinter fixes this performance issue and we can turn `errcheck` back on.